peps/pep-0787.rst

```diff
@@ -18,6 +18,18 @@ proposes extending the :mod:`subprocess` and :mod:`shlex` modules to natively su
 safer and more ergonomic shell command execution with interpolated values, as well as
 serving as a reference implementation for the t-string feature to improve API ergonomics.
 
+PEP Deferral
+============
+
+During the discussions of the initial draft of the PEP, it became clear that t-strings provide
+a potential opportunity to offer ``shell=True`` levels of syntactic convenience for complex
+subprocess invocations without all of the security and cross-platform compatibility concerns
+that arise with giving user provided text access to a full system shell.
+
+To that end, the PEP authors now plan to work on an experimental `t-string based subprocess
+invocation library <https://pypi.org/project/tstrprocess/>`__ through the Python 3.14 beta
```

Comment:
We can use the extlink role for this:

https://github.com/python/peps/blob/45e6128945c1e70cd9fed137e43f61f16ed42b46/peps/conf.py#L82-L88

```suggestion
To that end, the PEP authors now plan to work on an experimental :pypi:`t-string
based subprocess invocation library <tstrprocess>` through the Python 3.14 beta
```